Sage-Bionetworks/sage-monorepo
# [bug fix]: fixed Python version in api test

Fixed Python version in api test. 

## To test
1. `cd /apps/schematic/api`
2. `nx run test` and you should be able to see tests running.
